common/src/main/java/org/apache/celeborn/common/network/client/TransportResponseHandler.java

```diff
@@ -95,15 +95,15 @@ public TransportResponseHandler(TransportConf conf, Channel channel) {
         if (pushTimeoutChecker == null) {
           pushTimeoutChecker =
               ThreadUtils.newDaemonThreadPoolScheduledExecutor(
-                  "push-timeout-checker", conf.pushDataTimeoutCheckerThreads());
+                  "celeborn-client-push-timeout-checker", conf.pushDataTimeoutCheckerThreads());
```

Comment:
This thread is used by clients and workers.

Reply:
> This thread is used by clients and workers.

How about current?